### ISSUE TITLE
Perhaps a config file or override toggle?

### ISSUE THREAD

**Issue by lifehome** [dgraziotin/mbpfan#119]
**tl;dr** As long as the daemon is running, it seems that a manual override is impossible at all, as the daemon itself is constantly monitoring and sending a manual value to the SMC.

So perhaps a config file, or even better a "manual override" in case of contingency?

---
## What happened?
Just last Friday, my phone was ringing all around the noon, and a fatal alarm is triggered due to malfunction of cooling/airflow device, and high ambient temperature(31.57°C and 0.5°C increase/min). In addition to the incomplete design of my monitoring system, I have to login to Mac Mini (Late 2012, i5-3210M@2.5Ghz, 1 internal fan module) and tune the fan to a higher speed.
When I attempted the following command, the system shows the fan speed raise at the target level(5500rpm), then drops back to around 2000rpm within seconds.
```
# echo 1 > /sys/devices/platform/applesmc.768/fan1_manual
# echo $(cat /sys/devices/platform/applesmc.768/fan1_max) > /sys/devices/platform/applesmc.768/fan1_output
```
## What action has taken?
I have tried to `man` the mbpfan command, however it seems there are no manual override of the daemon, nor any command to instruct the SMC directly. Finally the manual command worked, removing the AUR package and killing the daemon.

**Comment by dgraziotin** [comment]
Thanks for the suggestion. Sounds like a nice improvement. Just to be sure. What you are asking is a way way for telling mpbfan to just sit there and do nothing instead of having to kill mbpfan. Did I get it right?

I am not sure about the fan*_manual thing. It is my understanding that mbpfan needs to do this (that is, claim manual control of fans) in order to have applesmc do nothing about them. Shall I be wrong about this, please do tell.

Another way would be to have a custom file to be touched somewhere to let mbpfan just..stay there.

**Comment by lifehome** [comment]
Thanks for considering this as an enhancement!

Sorry for the confusion, as the fact I am suggesting adding a command like `mbpfan --setspeed 5500` to lock fan speed until next reboot(reset to default), or another command.

On the other side, the configuration file suggests desired sets of temperature and fanspeed. Once the ambient or core or sensor temperature reach the configured level, mbpfan will set constant fanspeed accordingly.

**Comment by dgraziotin** [comment]
Ah, you mean to have mbpfan act as a wrapper for the "echo 5500 > /sys/devices/platform/applesmc.768/fan*_output" command, then? (and to have mpbfan just keep that value forever in the current session).

**Comment by lifehome** [comment]
Yes that's correct, since mbpfan will constantly set the fan speed value, it would be nice if mbpfan itself can set the speed value too.

**Comment by dgraziotin** [comment]
Hmm, that excites me less than a trigger to make mbpfan stay quiet. 

If you need to have mbpfan sent the fan speed manually, you still need to kill mbpfan and rerun it with the --setspeed 5500 command.

That is the same numbers of steps required to to it with the echo command. I do not see much of an improvement. Unless I am not getting something?

**Comment by dgraziotin** [comment]
closing this up to when it is clear that it is needed.
